.github/workflows/auth0-terraform-plan.yml

```diff
@@ -0,0 +1,72 @@
+name: "Auth0 Terraform Plan"
```

Comment:
Going to nudge for the convention of adding a ♻️ emoji to the name to highlight it's CICD 😉 also may be good to rename the file as `terraform-plan-auth0.yml`, so if/when we add more terraform, they're all be next to each other in the IDE 🦆

Reply:
I probably should add something to the readme so that we (mainly I) can actually see the convention 😂